2022-03-24 22:44:08,082 INFO    MainThread:21136 [wandb_setup.py:_flush():71] setting env: {}
2022-03-24 22:44:08,083 INFO    MainThread:21136 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-24 22:44:08,083 INFO    MainThread:21136 [wandb_init.py:_log_setup():371] Logging user logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220324_224408-3eowljts\logs\debug.log
2022-03-24 22:44:08,084 INFO    MainThread:21136 [wandb_init.py:_log_setup():372] Logging internal logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220324_224408-3eowljts\logs\debug-internal.log
2022-03-24 22:44:08,084 INFO    MainThread:21136 [wandb_init.py:init():404] calling init triggers
2022-03-24 22:44:08,084 INFO    MainThread:21136 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-24 22:44:08,094 INFO    MainThread:21136 [wandb_init.py:init():460] starting backend
2022-03-24 22:44:08,094 INFO    MainThread:21136 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-03-24 22:44:08,099 INFO    MainThread:21136 [backend.py:ensure_launched():216] starting backend process...
2022-03-24 22:44:08,159 INFO    MainThread:21136 [backend.py:ensure_launched():222] started backend process with pid: 21624
2022-03-24 22:44:08,160 INFO    MainThread:21136 [wandb_init.py:init():469] backend started and connected
2022-03-24 22:44:08,161 INFO    MainThread:21136 [wandb_init.py:init():533] updated telemetry
2022-03-24 22:44:08,373 INFO    MainThread:21136 [wandb_init.py:init():563] communicating current version
2022-03-24 22:44:10,059 INFO    MainThread:21136 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-24 22:44:10,060 INFO    MainThread:21136 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-24 22:44:10,955 INFO    MainThread:21136 [wandb_init.py:init():606] starting run threads in backend
2022-03-24 22:44:15,183 INFO    MainThread:21136 [wandb_run.py:_console_start():1810] atexit reg
2022-03-24 22:44:15,184 INFO    MainThread:21136 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-03-24 22:44:15,185 INFO    MainThread:21136 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-03-24 22:44:15,188 INFO    MainThread:21136 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-24 22:44:15,188 INFO    MainThread:21136 [wandb_init.py:init():633] run started, returning control to user process
2022-03-24 22:47:50,352 INFO    MainThread:21136 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-03-24 22:47:50,353 INFO    MainThread:21136 [wandb_run.py:_restore():1752] restore
2022-03-24 22:47:50,910 INFO    MainThread:21136 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-03-24 22:47:52,336 INFO    MainThread:21136 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-03-24 22:47:52,603 INFO    MainThread:21136 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 18887
}

